packages/markdown/processor/parse/flavored/callout.js

```diff
@@ -22,9 +22,7 @@ function tokenizer(eat, value) {
     '\uD83D\uDED1': 'error',
     '\u2049\uFE0F': 'error',
     '\u203C\uFE0F': 'error',
-    // NOTE: prettier is desperate to convert this in to an emoji.
```

Comment:
Can you leave this comment in?

Reply:
Assuming this updated `prettier-ignore` statement works I’m comfortable deleting my scary comment!

Reply:
Yeah, I just want to leave the scary comment so we know why the prettier ignore is there.